components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/psa_crypto_spm.c

```diff
@@ -140,6 +135,9 @@ psa_status_t psa_mac_update( psa_mac_operation_t *operation,
     psa_invec_t in_vec[2] = { { &psa_crypto_ipc, sizeof( psa_crypto_ipc ) },
                             { input, input_length } };
 
+    if( operation->handle <= 0 )
```

Comment:
Minor: when would `handle` be negative? Why isn't this `operation->handle == 0`?

Reply:
Handles returned from psa_connect() by value. Negative range and zero stands for errors.